client/verta/verta/_internal_utils/http_session_utils.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+""" Utilities for working with request.Session objects, and handling of retries """
+
+# TODO: Unit test for prediction_input_unpack method
+# TODO: Unit test for prediction_io_cleanup method
+import requests
+
+from requests import Session
+from requests.adapters import HTTPAdapter
+from typing import Dict, Optional, Set
+from urllib3.util import Retry
+
+DEFAULT_MAX_RETRIES: int = 13
+DEFAULT_BACKOFF_FACTOR: float = 0.3
+DEFAULT_STATUS_FORCELIST: Set = {404, 429}
+
+
+def retry_config(max_retries: int,
+                 status_forcelist: Set[int],
+                 backoff_factor: float) -> Retry:
```

Comment:
I think we need a consistent style for breaking function signatures into multiple lines.

I'm seeing that sometimes the `) -> RetType` is on the same line as the last parameter (like here), sometimes it's on the next line at the same indentation as the parameters (like [here for `DeployedModel.predict()`](https://github.com/VertaAI/modeldb/pull/3332/files#diff-43335cbae366a467266b824f5dc001bffc653b4f7fa813981b756cfcbc7bcb2bR164)), and sometimes it's on the next line at the same indentation as `def` (like [here for `DeployedModel.predict_with_id()`](https://github.com/VertaAI/modeldb/pull/3332/files#diff-43335cbae366a467266b824f5dc001bffc653b4f7fa813981b756cfcbc7bcb2bR240)).